sass/sass_wrapper.js

```diff
@@ -0,0 +1,46 @@
+const {debug, log, runAsWorker, runWorkerLoop} = require('./worker');
+const sass = require('sass');
+const fs = require('fs');
+
+// Equivalent of running node with --expose-gc
+// but easier to write tooling since we don't need to inject that arg to
+// nodejs_binary
+if (typeof global.gc !== 'function') {
+  require('v8').setFlagsFromString('--expose_gc');
+  global.gc = require('vm').runInNewContext('gc');
+}
+
+function main(args) {
+  if (runAsWorker(args)) {
+    log('Starting Sass compiler persistent worker...');
+    runWorkerLoop(runOneBuild);
+    // Note: intentionally don't process.exit() here, because runWorkerLoop
+    // is waiting for async callbacks from node.
+  } else {
+    debug('Running a single build...');
+    if (args.length === 0) throw new Error('Not enough arguments');
+    if (args.length !== 1) {
+      console.error('Expected one argument: path to flagfile');
+      return 0;
+    }
+    const configFile = args[0].replace(/^@+/, '');
```

Comment:
Add a comment that explains why this `replace` occurs?

Reply:
Done.